yt/visualization/volume_rendering/transfer_function_helper.py

```diff
@@ -190,9 +188,8 @@ def plot(self, fn=None, profile_field=None, profile_weight=None):
                 prof[profile_field]
             except KeyError:
                 prof.add_fields([profile_field])
-            # Strip units, if any, for matplotlib 1.3.1
-            xplot = np.array(prof.x)
-            yplot = np.array(
+            xplot = prof.x
```

Comment:
We could probably just dump `prof.x` into the call to the plotting function, but it's probably clearer to leave like this.